addons/resource/models/resource_mixin.py

```diff
@@ -33,17 +33,18 @@ class ResourceMixin(models.AbstractModel):
         string='Timezone', related='resource_id.tz', readonly=False,
         help="This field is used in order to define in which timezone the resources will work.")
 
-    @api.model
-    def create(self, values):
-        if not values.get('resource_id'):
-            resource_vals = {'name': values.get(self._rec_name)}
-            tz = (values.pop('tz', False) or
-                  self.env['resource.calendar'].browse(values.get('resource_calendar_id')).tz)
-            if tz:
-                resource_vals['tz'] = tz
-            resource = self.env['resource.resource'].create(resource_vals)
-            values['resource_id'] = resource.id
-        return super(ResourceMixin, self).create(values)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if not vals.get('resource_id'):
+                resource_vals = {'name': vals.get(self._rec_name)}
+                tz = (vals.pop('tz', False) or
+                      self.env['resource.calendar'].browse(vals.get('resource_calendar_id')).tz)
```

Comment:
Seems we are doing unnecessary browse, as we have no resource calendar to browse due to the "if"
```
if not vals.get('resource_id')
```
and we do
```
self.env['resource.calendar'].browse(vals.get('resource_calendar_id')).tz)
```

-> seems we could end up on calendar default directly which is ``self._context.get('tz') or self.env.user.tz or 'UTC'``